galsim/fits.py

```diff
@@ -537,6 +540,10 @@ def write(image, file_name=None, dir=None, hdu_list=None, clobber=True, compress
     """
     from galsim._pyfits import pyfits
 
+    if image.iscomplex:
```

Comment:
I propose making a separate issue for this.  Mostly since I'm not sure what the use case is for writing a complex image to a fits file, so I'm not sure what the most appropriate way to write it would be from a user-friendliness standpoint.  For instance another option would be to write the real and imaginary components to separate HDUs, which would let them be easily viewable by ds9.  But that complicates some of the hdu bookkeeping we currently have.

So I think it would be worth making an issue and seeing if we get some ideas about how people would like it to work before choosing an implementation.